include/modem/lte_lc.h

```diff
@@ -214,9 +317,161 @@ enum lte_lc_pdp_type {
 };
 
 enum lte_lc_pdn_auth_type {
-	LTE_LC_PDN_AUTH_TYPE_NONE = 0,
-	LTE_LC_PDN_AUTH_TYPE_PAP = 1,
-	LTE_LC_PDN_AUTH_TYPE_CHAP = 2,
+	LTE_LC_PDN_AUTH_TYPE_NONE	= 0,
+	LTE_LC_PDN_AUTH_TYPE_PAP	= 1,
+	LTE_LC_PDN_AUTH_TYPE_CHAP	= 2,
+};
+
+enum lte_lc_energy_estimate {
+	LTE_LC_ENERGY_CONSUMPTION_EXCESSIVE	= 5,
+	LTE_LC_ENERGY_CONSUMPTION_INCREASED	= 6,
+	LTE_LC_ENERGY_CONSUMPTION_NORMAL	= 7,
+	LTE_LC_ENERGY_CONSUMPTION_REDUCED	= 8,
+	LTE_LC_ENERGY_CONSUMPTION_EFFICIENT	= 9,
+};
+
+enum lte_lc_tau_triggered {
+	LTE_LC_CELL_IN_TAI_LIST		= 0,
+
+	/** Cell not in TAI list, TAU will be triggered. */
+	LTE_LC_CELL_NOT_IN_TAI_LIST	= 1,
+	LTE_LC_CELL_UNKNOWN		= UINT8_MAX
+};
+
+enum lte_lc_ce_level {
+	LTE_LC_CE_LEVEL_0_NO_REPETITION		= 0,
+	LTE_LC_CE_LEVEL_1_LOW_REPETITION	= 1,
+	LTE_LC_CE_LEVEL_2_MEDIUM_REPETITION	= 2,
+	LTE_LC_CE_LEVEL_UNKNOWN			= UINT8_MAX,
+};
+
+/** @brief Connection evaluation parameters.
+ *
+ *  @note For more information on the various connection evaluation parameters, refer to the
+ *	  "nRF91 AT Commands - Command Reference Guide".
+ */
+struct lte_lc_conn_eval_params {
+	/** RRC connection state during measurements. */
+	enum lte_lc_rrc_mode rrc_state;
+
+	/** Relative estimated energy consumption for data transmission compared to nominal
+	 *  consumption.
+	 */
+	enum lte_lc_energy_estimate energy_estimate;
+
+	/** Is the evaluated cell part of the current TAI list. */
```

Comment:
What's the TAI list? I think we should open this acronym. I'll try to figure out what it is...

Reply:
TAI means Tracking Area Identifier. This list is got from the network and if a cell is on that list, no TAU is needed when switching to that cell.

Reply:
I was wondering the exact same thing. Thanks for the explanation, will update.

Reply:
Added additional description to both the variable and enums.